tests/unit/lms/resources/_lti_launch_test.py

```diff
@@ -153,7 +153,8 @@ def test_h_display_name(
     def test_h_authority_provided_id_raises_if_theres_no_tool_consumer_instance_guid(
```

Comment:
We are now picky about values we weren't before. All of these tests we starting with empty params and adding a few things. Now we start with default params and remove them or change them as required.

I think this actually reads better too, as quite often the test is something like "when_X_missing" and then never mentions "X". Now a test like that will remove "X" as it's first order of business.